qa/suites/orch/cephadm/mgr-nfs-upgrade/1-start.yaml

```diff
@@ -1,3 +1,9 @@
+overrides:
+  ceph:
+    log-ignorelist:
+      - MON_DOWN
+      - PG_DEGRADED
```

Comment:
still hit a failure for PG_DEGRADED in the logs of this test
https://pulpito.ceph.com/adking-2024-02-27_05:08:20-orch:cephadm-wip-adk-testing-2024-02-26-1600-distro-default-smithi/7574463
it seems like none of these log-ignorelist entries actually made there way into the generated yaml

Reply:
overrides came out like
```
overrides:
    admin_socket:
      branch: wip-adk-testing-2024-02-26-1600
    ceph:
      conf:
        mgr:
          debug mgr: 20
          debug ms: 1
        mon:
          debug mon: 20
          debug ms: 1
          debug paxos: 20
        osd:
          debug ms: 1
          debug osd: 20
          osd shutdown pgref assert: true
      flavor: default
      log-ignorelist:
      - \(MDS_ALL_DOWN\)
      - \(MDS_UP_LESS_THAN_MAX\)
      sha1: 635ee6850c57c98cc6ba932cade45f9cc015aa32
```
I have no idea where it's getting only
```
      - \(MDS_ALL_DOWN\)
      - \(MDS_UP_LESS_THAN_MAX\)
```
from. I don't see any ignorelists with only those two anywhere